src/webapp/azext_webapp/_help.py

```diff
@@ -59,3 +59,8 @@
         - name: Upload files from the current directory to an Azure Container Registry, then build a container image and deploy it to a web app. The Azure Container Registry must already exist.
           text: az webapp container up -n AppName --registry-rg ContainerRegistryResourceGroup --registry-name ContainerRegistryName
 """
+
+helps['webapp deploy'] = """
+    type: command
```

Comment:
Can you add some examples here for different types and if the path can be relative vs absolute path?